features/sign_up_page.feature

```diff
@@ -9,24 +9,24 @@ Scenario: Submit sign up form [Happy path]
     And I fill in "Email" with "camron@heroku.com"
     And I fill in "Password" with "camron1234"
     And I fill in "Password confirmation" with "camron1234"
-    When I click "Create" to submit form
+    When I click "Create" 
     And I should see "Welcome! You have signed up successfully."
 
 Scenario: Submit sign up form without name [Sad path]
     When I visit the "sign up" page
     Then I fill in "Name" with " "    
-    When I click "Create" to submit form
+    When I click "Create" 
     Then I should see "Name can't be blank"
 
 Scenario: Submit sign up form without email [Sad path]
     When I visit the "sign up" page
     Then I fill in "Email" with " "
-    When I click "Create" to submit form
+    When I click "Create" 
     Then I should see "Email can't be blank"
  
 Scenario: Submit sign up form without password [Sad path]
     When I visit the "sign up" page
     Then I fill in "Password" with " "
-    When I click "Create" to submit form
+    When I click "Create" 
```

Comment:
Why are you adding the `"Create"` in the signup feature file?
Please explain the reason here.

Reply:
1. We did not add but modified that line
2. The submit button on the sign up page is called Create

Reply:
done